### DISCUSSION TITLE
Remove Race Control Messages of the Live Timing when the RCM page is open.

### DISCUSSION THREAD

**Discussion by LapsTimeOFF** [f1multiviewer/issue-tracker#139] [Feature requests] [answered]
I think it will be better if when we click on the  <img width="137" alt="image" src="https://user-images.githubusercontent.com/39061148/181918555-22b5526d-1071-4891-84a0-e6c14452dcf4.png"> button the RCM part get removed in the Live Timing page. It will be great because we can better see the map.

**Comment by JJWatMyself** [comment]
I agree that toggle key for RCM off and also option for pop-out track map (toggle M for track map already exists).

**Comment by Sloth-on-meth** [comment]
so 

 - remove RCM from main live timing window when popout is in use
 - add popout for track map 
 
 +1

**Reply by LapsTimeOFF** [reply]
For Add popout for track map look #138

**Comment by f1multiviewer** [comment, marked as answer]
You can now do this with a setting!
